easybuild/easyconfigs/b/BuildEnv/BuildEnv-1.0-intel-2015a-default.eb

```diff
@@ -0,0 +1,14 @@
+easyblock = "BuildEnv"
+
+name = 'BuildEnv'
+version = '1.0'
```

Comment:
Originally I wanted to use ```default``` as the version but lmod doesn't like that, it only wants to see things that begin with numbers so I was forced to add a version number